calico/common.py

```diff
@@ -71,18 +75,27 @@
     "protocol",
     "src_net",
     "src_tag",
+    "src_selector",
     "src_ports",
     "dst_net",
     "dst_tag",
+    "dst_selector",
     "dst_ports",
     "icmp_type",
     "icmp_code",
     "ip_version",
 ])
 
+# Valid actions to see in a rule.
+KNOWN_ACTIONS = set(["allow", "deny", "next-tier"])
```

Comment:
Only consistency at this point, I think.  I'd rather make a single change that swaps `set([...])` for `{...}` everywhere if we're going to do that.